src/SAWScript/Interpreter.hs

```diff
@@ -2616,6 +2634,16 @@ primitives = Map.fromList
     Current
     [ "Use the default set of callee-saved registers during x86 verification." ]
 
+  , prim "enable_what4_eval" "TopLevel ()"
```

Comment:
Can you add a section to the changelog describing the new `{enable,disable}_what4_eval` functions?